### PR TITLE
Add ci build support for Travis

### PR DESCRIPTION
**Pull request by jdcba**
The only issue here is that it involves adding a ci repo tarball. ci is currently a private repo. Is this a problem?

I also added Maven Central for required dependencies that are missing from Bintray.

### CODE REVIEW

**Comment by rowandavies** [comment]
ci was public in the past - since many public repos depended on it.

File: .travis.yml [hunk 1]
```diff
@@ -7,7 +7,8 @@ cache:
   - $HOME/.ivy2
   - $HOME/.m2
 install:
-- git clone https://github.com/CommBank/ci.git
+# git clone https://github.com/CommBank/ci.git
+- tar xzf .ci-a53f524.tgz # git archive -o ci-a53f524.tgz --prefix=ci/ master
```
Comment (rowandavies) [inline .travis.yml]: Maybe add a comment here on why we have this workaround?

**Comment by jdcba** [comment]
@SamRoberts your thoughts: would you like to make ci public again (I can’t), or accept the bundling that I did here? In future we would migrate to ci2, but it’s not trivial and is blocking the CDH testing and has been resistant to a solution.

**Comment by rowandavies** [comment]
Approving subject to input from @SamRoberts

**Comment by SamRoberts** [comment]
Hmm, ci was private before it was deleted, Are you sure it had to be public @rowandavies?

I am happy in principle to just make it public, but I'll need to take a look at the repo first.

**Comment by SamRoberts** [comment]
Ah yes, I can see the commit from when I added licenses to a ton of repos we were making public: https://github.com/CommBank/ci/commit/a22efb01d5ae3030883c52a9553db44e3f4bdd19

**Comment by jdcba** [comment]
Hi Sam yes it was public. My understanding is that it was turned private when ‘superseded’ by ci2.

**Comment by SamRoberts** [comment]
Damn that's a lot of bash.

Ok, we need an apache header for:

1. dockermake.sh
2. gh-commit-comment.sh
3. do test.bsh and tests folder scripts need an apache header? might just add one ...

Will references to `travis-ci.org` need to be changed? I guess we'll find out ...

I'll submit a PR on ci to add those headers and make it public after they are in.

**Comment by SamRoberts** [comment]
Could the non-specific "you" please review https://github.com/CommBank/ci/pull/1/files in case I've missed something?

**Comment by SamRoberts** [comment]
Alright, ci is public :)

**Pull request merged**